README.md

```diff
@@ -15,7 +15,7 @@ The rules can be run manually or automatically by a job which is based on OpenMR
   * The following is a sample insert command.
 
 ```sql
-insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.dataintegrity.InvalidDateOfDeath');
+insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.basicexample.dataintegrity.InvalidDateOfDeath');
```

Comment:
@ssmusoke , okay I'll update it. 
I guess there is already a similar issue in the JIRA: https://issues.openmrs.org/browse/DINT-8